src/main/java/hudson/plugins/git/GitSCM.java

```diff
@@ -564,8 +564,6 @@ private PollingResult compareRemoteRevisionWithImpl(Job<?, ?> project, Launcher
 
         final EnvVars pollEnv = project instanceof AbstractProject ? GitUtils.getPollEnvironment((AbstractProject) project, workspace, launcher, listener, false) : lastBuild.getEnvironment(listener);
 
-        final String singleBranch = getSingleBranch(pollEnv);
-
```

Comment:
Is the move of this line actually changing behavior (say, because `pollEnv` was modified in the intervening lines), or merely a stylistic change to move the definition closer to its usage?